opentelemetry-otlp/src/logs.rs

```diff
@@ -107,106 +132,3 @@ impl opentelemetry_sdk::export::logs::LogExporter for LogExporter {
         self.client.set_resource(resource);
     }
 }
-
-/// Recommended configuration for an OTLP exporter pipeline.
```

Comment:
We'll need to make a detailed changelog entry as this is breaking change. I think its best to include before/after code snippets in changelog itself for some common scenario, and a link to the PR.
This PR's description can contain the full details and a mini migration guide.